pkg/kubectl/cmd/apply_test.go

```diff
@@ -1121,7 +1121,7 @@ func TestRunApplySetLastApplied(t *testing.T) {
 			name:        "set with exist object",
 			filePath:    filenameRC,
 			expectedErr: "",
-			expectedOut: "replicationcontroller/test-rc configured\n",
```

Comment:
This looks suspicious. Why did this change?

Reply:
The tests set the output format `name`, so instead of printing the success output (which includes "configured"), only the resource name is printed.

This was passing before because I was not calling PrintFlags.AddFlags from the set_last_applied command